Websites/webkit.org/demos/webgpu/two-pass-instanced-textured-cube.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Tests are very high priority! I think ref-tests were suggested possibly? I filed a bug for unit tests. While important, I don't think unit tests would go far in helping is to actually validate functionally things are working.

I just need to figure out how to setup ref-tests or any other suggested testing method - suggestions / pointers are welcomed

Reply:
I think we should use _only_ the CTS for testing. If there's a test that we want to write that isn't in the CTS, we should write it and upstream it.

Reply:
this is the CTS right? https://gpuweb.github.io/cts/standalone/?q=webgpu:*

Reply:
I don't think we can get to a state where we can only use CTS tests or only upstream CTS tests until we have made more progress on the compiler, since I don't think we can submit a CTS test that has metal source